src/cljs/witan/ui/components/icons.cljs

```diff
@@ -139,6 +142,50 @@
   [& args]
   (apply create-key "play_arrow" args))
 
+(defn bug
+  [& args]
+  (apply create-key "bug_report" args))
+
+(defn request-to-share
+  [& args]
+  (apply create-key "assignment_return" args))
+
+(defn user
+  [& args]
+  (apply create-key "person" args))
+
+(defn organisation
+  [& args]
+  (apply create-key "location_city" args))
+
+(defn delete
+  [& args]
+  (apply create-key "delete" args))
+
+(defn tick
+  [& args]
+  (apply create-key "done" args))
+
+(defn schema
+  [& args]
+  (apply create-key "assignment" args))
+
+(defn email
+  [& args]
+  (apply create-key "email" args))
+
+(defn tick-circle
+  [& args]
+  (apply create-key "check_circle" args))
+
+(defn warning
```

Comment:
Not entirely sure what all these little fn's are for, but it feels like it would be clearer if the names of the functions matched the key they created.